### PR TITLE
Add release notes section for 0.9.2

### PR DESCRIPTION
**Pull request by mjdesa**
Looking for more things to add here.

### CODE REVIEW

File: content/docs/v0.9/about/release_notes.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 title: Release Notes
 ---
 
+## v0.9.2
+
+### Update notes
```
Comment (beckettsean) [inline content/docs/v0.9/about/release_notes.md]: Should this be `Upgrade notes` for consistency?

Comment (mjdesa) [inline content/docs/v0.9/about/release_notes.md]: Yeah totally. Temporary brain malfunction.

File: content/docs/v0.9/about/release_notes.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
 title: Release Notes
 ---
 
+## v0.9.2
+
+### Upgrade notes
+
+- [BREAKING CHANGE] `SHOW RETENTION POLICIES mydb` is  now `SHOW RETENTION POLICIES ON mydb`. [3345](https://github.com/influxdb/influxdb/pull/3345)
```
Comment (gunnaraasen) [inline content/docs/v0.9/about/release_notes.md]: This is a bug fix and should be mentioned under bugfixes. Also not sure if it warrants the `BREAKING CHANGE` billing.

Comment (mjdesa) [inline content/docs/v0.9/about/release_notes.md]: kk

Comment (beckettsean) [inline content/docs/v0.9/about/release_notes.md]: I think leave the breaking change but let's move it to bug fixes.

**Comment by beckettsean** [comment]
LGTM, merging

**Pull request merged**